main-app/app/less/styles.less

```diff
@@ -14,11 +14,11 @@ table {
 }
 
 .nought {
-  background: url("../images/nought.png") 0 0;
-  background-size: (@squareWidth + 1) (@squareHeight + 1);
+  background: url("../images/noughtAndCrossSheet.png") 101px 0;
```

Comment:
Why is this 101px? You should also be able to use the @squareWidth 
Remember whitespace between tags is important - it renders as a space and this can knock the formatting out.
Other culprits include margins / borders and content spilling over.